awx/main/migrations/0001_initial.py

```diff
@@ -380,7 +380,7 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='AdHocCommand',
             fields=[
-                ('unifiedjob_ptr', models.OneToOneField(parent_link=True, auto_created=True, primary_key=True, serialize=False, to='main.UnifiedJob')),
+                ('unifiedjob_ptr', models.OneToOneField(parent_link=True, auto_created=True, primary_key=True, on_delete=django.db.models.deletion.CASCADE, serialize=False, to='main.UnifiedJob')),
```

Comment:
I'm seeing a lot of new cascades on this initial PR - including users.  This new schema _won't_ be captured on upgrade, because people will have already migrated past this.  Is that going to be an issue?  What's the Django change that necessitates this change?

Reply:
`OneToOneField` and `ForeignKey` now require the `on_delete` parameter as of Django 2.0.  Even when it is used in a migration.  The default behavior before was a cascading delete, so most of the changes I made were to make that explicit.

https://docs.djangoproject.com/en/2.2/releases/2.0/#features-removed-in-2-0

Reply:
But it sounds like the cascade is still accomplished in Python, not via a database-backed cascade.  If that's the case (sounds like it is), I'm good with this 👍